shaded-ning19/pom.xml

```diff
@@ -23,13 +23,14 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>org.apache.apex</groupId>
-    <artifactId>apex</artifactId>
-    <version>3.3.0-incubating</version>
+    <groupId>org.apache</groupId>
+    <artifactId>apache</artifactId>
+    <version>18</version>
+    <relativePath/>
   </parent>
   <groupId>org.apache.apex</groupId>
   <artifactId>apex-shaded-ning19</artifactId>
-  <version>1.0.0</version>
+  <version>1.1.0-SNAPSHOT</version>
```

Comment:
What is the plan for the next Apex core release? Is it planned to go out with SNAPSHOT in the apex-shaded-ning19 pom.xml? I guess another option will be to exclude apex-shaded-ning19 and codestyle-config from source release distribution.